flink-libraries/flink-table/src/main/scala/org/apache/flink/table/descriptors/DescriptorProperties.scala

```diff
@@ -858,13 +859,18 @@ class DescriptorProperties(normalizeKeys: Boolean = true) {
   /**
     * Validates a type property.
     */
-  def validateType(key: String, isOptional: Boolean): Unit = {
+  def validateType(key: String, requireRow: Boolean, isOptional: Boolean): Unit = {
     if (!properties.contains(key)) {
       if (!isOptional) {
         throw new ValidationException(s"Could not find required property '$key'.")
       }
     } else {
-      TypeStringUtils.readTypeInfo(properties(key)) // throws validation exceptions
+      // throws validation exceptions
```

Comment:
what's the purpose of this comment?

Reply:
That we don't validate the string but let the parser do the work for us. I updated the comment.